src/main/java/uk/gov/ch/pocappointmentapi/AppointmentController.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.ch.pocappointmentapi;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AppointmentController {
+
+	@Autowired
+	AppointmentRepository repository;
+
+	@GetMapping(value = "/company/{companyNumber}/appointment/{appointmentId}")
+	public Appointment appointment(@PathVariable String companyNumber, @PathVariable String appointmentId) {
+		return repository.findByCompanyNumberAndAppointmentId(companyNumber, appointmentId);
```

Comment:
We would have to add another layer of abstraction here (e.g. Service object) to support transformation.

Reply:
Yes, in the actual implementation we would need a service layer - but not needed for POC.